src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{std::move(ca), std::move(ref)}
```

Comment:
I use MinGW-W64 GCC-7.1.0 for build my project.
https://sourceforge.net/projects/mingw-w64/files/